CHANGES.md

```diff
@@ -33,6 +33,8 @@ Notable changes between versions.
 
 ### Fedora CoreOS
 
+* Fix SELinux label of bootstrap-secrets on non-bootstrapping controllers ([#808](https://github.com/poseidon/typhoon/pull/808))
```

Comment:
can you move to latest, this section is for v1.18.8

Reply:
@dghubble I thought v1.18.8 on the same level with Fedora CoreOS since they are both prefixed with ###. It turns out that v1.18.8 should be prefixed with ##